CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Change Log
 
+## 0.19.0 - 2020-09-15
+### Added
+- Connections support to PostgreSQL accounts through Azure Databases extension
```

Comment:
I don't know why, but your existing sentence just sounds awkward to me. How about this

```suggestion
- Connect a PostgreSQL server to your web app through the [Azure Databases extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-cosmosdb)
```